articles/logs/streams/datadog.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Raising this PR for initial feedback, but we will probably merge it in a week or so when the Datadog team is ready on their side with the integration:

https://docs-content-staging-pr-8792.herokuapp.com/docs/logs/streams/datadog